Source/Editor/DualityEditor/Utility/DualityEditorUserData.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Xml.Linq;
+using Duality.Resources;
+
+namespace Duality.Editor
+{
+	public class DualityEditorUserData
+	{
+		private DesignTimeObjectDataManager designTimeObjectDataManager = new DesignTimeObjectDataManager();
```

Comment:
As per style guide, please group all fields together at the top, followed by the equally grouped properties.